packages/ui/components/TokenHandler/tokenhandler.stories.mdx

```diff
@@ -3,6 +3,7 @@ import { Examples, Example, Note, Title,CustomArgsTable } from '@calcom/storyboo
 import TokenHandler from './TokenHandler'
 
 <Meta title="UI/TokenHandler" component={TokenHandler} />
+
```

Comment:
[Here's the culprit](https://github.com/storybookjs/storybook/issues/10367#issuecomment-612349599)